instaloader/instaloadercontext.py

```diff
@@ -211,6 +211,9 @@ def save_session(self):
         """Not meant to be used directly, use :meth:`Instaloader.save_session`."""
         return requests.utils.dict_from_cookiejar(self._session.cookies)
 
+    def update_cookie(self, cookie):
```

Comment:
I would call it `update_cookies` (plural) since it actually updates multiple cookies.

Reply:
Updated